tests/__init__.py

```diff
@@ -36,6 +39,12 @@
 _LOADER = botocore.loaders.Loader()
 
 
+def requires_memory_collection(cls):
```

Comment:
Can we indicate that this is a skip if decorator? Maybe rename it to ``skip_if_requires_memory_collection``